examples/cis3-nft-sponsored-txs/Cargo.toml

```diff
@@ -6,15 +6,17 @@ edition = "2021"
 license = "MPL-2.0"
 
 [features]
-default = ["std", "wee_alloc", "crypto-primitives"]
+default = ["std", "wee_alloc"]
```

Comment:
We still use the attribute `crypto-primitives` for some functions, yes, but we do not need this feature anymore.
With the `test_infrastructure` you use mocks for all the crypto operations by default, but you can use real operations via the `concordium-std/crypto-primitives` feature.
The new testing lib always uses the real crypto operations 😊

Reply:
That'll also help with the reproducible builds thing.

Having to support "crypto-primitives" is annoying since a lot of extra dependencies need to be installed there.